spec/list-spec.coffee

```diff
@@ -88,6 +88,17 @@ describe 'apm list', ->
     listPackages [], ->
       expect(console.log.argsForCall[1][0]).toContain 'test-module@1.0.0 (disabled)'
 
+  it 'displays only enabled packages --enabled is called', ->
```

Comment:
Sorry about that... Should be fixed now as well as a layer of verification that the package field is not always empty.